samples/csharp_dotnetcore/14.nlp-with-dispatch/Startup.cs

```diff
@@ -64,7 +60,7 @@ public void ConfigureServices(IServiceCollection services)
 
             // Loads .bot configuration file and adds a singleton that your Bot can access through dependency injection.
             var botConfig = BotConfiguration.Load(botFilePath ?? @".\BotConfiguration.bot", secretKey);
-            services.AddSingleton(sp => botConfig ?? throw new InvalidOperationException($"The .bot configuration file could not be loaded. botFilePath: {botFilePath}"));
+            services.AddSingleton(sp => botConfig ?? throw new InvalidOperationException($"The .bot configuration file could not be loaded. ({botConfig})"));
```

Comment:
This was changed to `botFilePath: {botFilePath}` since the `({botConfig})` would be null at this point, should it be changed back?

Reply:
Indeed